SimTKcommon/Simulation/include/SimTKcommon/internal/MeasureImplementation.h

```diff
@@ -1289,6 +1289,43 @@ template <class T>
 class Measure_Differentiate_Result {
 public:
     Measure_Differentiate_Result() : derivIsGood(false) {}
+
+    Xml::Element toXmlElement(const std::string& name) const {
+        static const int version = 1;
+        Xml::Element e("Measure_Differentiate_Result");
+        if (!name.empty()) e.setAttributeValue("name", name);
+        e.setAttributeValue("version", String(version));
+        e.appendNode(toXmlElementHelper(operand, "operand", true));
+        e.appendNode(toXmlElementHelper(operandDot, "operandDot", true));
+        e.appendNode(toXmlElementHelper(derivIsGood, "derivIsGood", true));
```

Comment:
I looked at this issue briefly. Maybe try the following:
* Free *template* function `toXmlElement<>()` in the namespaces of all classes that intend to support this method. 
* The primary template will have an implementation to just forward arguments to the member function of the class.
* This free function `toXmlElement()` may be specialized to change the behavior if desired.

This free function will be found through ADL (Argument Dependent Lookup) when a class is provided as the argument. Callers need not explicitly qualify the free function with the namespace. If a specialization exists for the class, it will be used. Or the primary template is chosen, which calls the member function.